gooddata-sdk/gooddata_sdk/compute/model/filter.py

```diff
@@ -18,6 +19,17 @@
 from gooddata_sdk.compute.model.base import Filter, ObjId
 from gooddata_sdk.compute.model.metric import Metric
 
+_DATE_FORMAT_INPUT = "%Y-%m-%d"
+_DATE_FORMAT_OUTPUT = "%-m/%-d/%Y"
```

Comment:
What do the `%-m` and `%-d` placeholders mean? I don't see them in the docs: https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes

Reply:
it's documented even deeper :D 
https://manpages.debian.org/bookworm/manpages-dev/strftime.3.en.html#glibc_notes